docs/content/modeling/organization-context-authorization.mdx

```diff
@@ -351,19 +356,17 @@ The "organization" type definition then becomes:
   }}
 />
 
-:::note
-On the "project" type:
+2. On the "project" type
 
 - Nothing will need to be done, as it will inherit the updated "project_manager" and "project_editor" relation definitions from "organization"
-:::
 
-##### Add The Required Tuples To Mark That anne Is In An Approved Context
+##### Add The Required Tuples To Mark That Anne Is In An Approved Context
 
-Now that we have updated our authorization model to take the current user's organization context into consideration, you will notice that anne has lost access because nothing indicates that anne is authorizing from the context of an organization. You can verify that by issuing the following check:
+Now that we have updated our authorization model to take the current user's organization context into consideration, you will notice that Anne has lost access because nothing indicates that anne is authorizing from the context of an organization. You can verify that by issuing the following check:
```

Comment:
```suggestion
Now that we have updated our authorization model to take the current user's organization context into consideration, you will notice that Anne has lost access because nothing indicates that Anne is authorizing from the context of an organization. You can verify that by issuing the following check:
```